arcade/hitbox/base.py

```diff
@@ -107,11 +184,30 @@ def create_rotatable(
         self,
         angle: float = 0.0,
     ) -> RotatableHitBox:
+        """
+        Create a rotatable instance of this hit box.
+
+        The internal point list is transferred directly instead of
+        deepcopied, so care should be taken if using a mutable internal
+        representation.
```

Comment:
I'm being super-nitpicky, but "internal point list" is slightly incorrect if it's a tuple, not a list.  Maybe something like "The internal point sequence is referenced directly instead of copied"?  I dunno.

Reply:
I don't like it either, but I don't want to rename the `PointList` type in this PR. I'm trying to be very disciplined about PR scope to build good habits & build up a good set of examples for the beginners.

Although the change could go in a prequel PR this one merges or rebases onto, I think it should be in a follow-up one. I don't want to start changing things einarf and Clepto understand the full implications of far better than I do without consulting them first.

Reply:
Oh I see, if it said `internal PointList is` then I would have understood what you were going for.  Seeing it as `point list` threw me off.